arbitrum-docs/partials/_glossary-partial.md

```diff
@@ -57,14 +63,23 @@
 ### Blockchain {#blockchain}
 <p>A distributed digital ledger that is used to record transactions and store data in a secure, transparent, and tamper-resistant way, notably in cryptocurrency protocols. </p>
 
+### BLS Signature {#bls-signature}
+<p>A cryptographic scheme that allows multiple signatures to be aggregated and compacted into one efficiently verifiable, constant-sized signature. Used in the <a href="/intro/glossary#arbitrum-anytrust-protocol">Arbitrum AnyTrust Protocol</a> for the <a href="/intro/glossary#data-availability-committee-dac">Data Availability Committee (DAC)</a>'s signatures.</p>
+
 ### BOLD {#bold}
 <p>Short for "Bounded Liquidity Delay"; latest version of the Arbitrum <a href="/intro/glossary#challenge-protocol">Challenge protocol</a> designed to eliminate <a href="https://medium.com/offchainlabs/solutions-to-delay-attacks-on-rollups-434f9d05a07a">delay attack vectors</a> (see <a href="https://medium.com/offchainlabs/bold-permissionless-validation-for-arbitrum-chains-9934eb5328cc">here</a> for more). Not currently on mainnet. </p>
 
+### Bridge {#bridge}
+<p>A set of smart contracts for sending <a href="/intro/glossary#crosschain-message">Cross-chain message</a>s between blockchains. Every <a href="/intro/glossary#arbitrum-chain">Arbitrum chain</a> includes a bridge to/from its <a href="/intro/glossary#parent-chain">Parent chain</a>.  </p>
+
 ### 🧢 CAP Finance {#cap-finance}
 <p>Widely considered the most degen community on Arbitrum.<br />
 <br />
 <a href="https://cap.io/">CAP</a> is a decentralized trading protocol. It's designed to be fast and easy to use by anyone.</p>
 
+### Chain Owner {#chain-owner}
+<p>An entity (i.e., a smart contract) with affordance to carry out critical upgrades to an Arbitrum chain's core protocol; this includes upgrading protocol contracts, setting core system parameters, and adding & removing other chain owners.</p>
```

Comment:
```suggestion
<p>An entity (i.e., a smart contract) with affordance to carry out critical upgrades to an Arbitrum chain's core protocol; this includes upgrading protocol contracts, setting core system parameters, and adding and removing other chain owners.</p>
```

Should we change the symbol for the word? Not sure if we have a pattern for that or not, but flagging just in case.

Reply:
@TucksonDev [the style guide says yes](https://docs.arbitrum.io/for-devs/contribute#style-conventions) :)